docs/source/serving/sky-serve.rst

```diff
@@ -521,3 +521,73 @@ The :code:`resources` field has the same spec as a normal SkyPilot job; see `her
   These settings will not take effect if you have an existing controller (either
   stopped or live).  For them to take effect, tear down the existing controller
   first, which requires all services to be terminated.
+
+Speedup Setup in Model Serving
+------------------------------
+
+When serving AI models, the setup process like dependencies installation takes a lot of time on the cold-start time. According to one experiment we found that, dependencies installation can take 45% of times (271s out of 608s) of the setup process. To speed up this process, you can use the following methods:
```

Comment:
Explicitly mention downloading model weights.  It's not commonly thought as being the same as dependency installation.